app/src/main/java/com/example/giftly/CThemeScreen.java

```diff
@@ -28,13 +30,14 @@ protected void onCreate(Bundle savedInstanceState) {
         int savedColor = sharedPreferences.getInt("BackgroundColor", ContextCompat.getColor(CThemeScreen.this, R.color.Default_color));
         getWindow().getDecorView().setBackgroundColor(savedColor);
 
-        //Create theme buttons
+        //Create variables to connect to xml android:id fields
         Button dftThemeBtn = findViewById(R.id.defaultTheme);
```

Comment:
Maybe Rename these to more vernacular names as we are no longer using holidays. maybe just use the color names?

Reply:
I will rename them by how they appear in the theme settings screen